### PR TITLE
Windows specific branch for configs

### PR DESCRIPTION
**Pull request by cyberious**

### CODE REVIEW

**Comment by mhaskel** [comment]
Why are you removing contributing.md?

**Comment by cyberious** [comment]
Not used on windows modules, if we want to revert having that I have no problems

**Comment by mhaskel** [comment]
It's generally good practice so I'd vote for putting it back and having it added to windows modules.

**Comment by cyberious** [comment]
@mhaskel added contributing back

**Comment by mhaskel** [comment]
:+1:

File: config_defaults.yml [hunk 1]
```diff
@@ -2,26 +2,44 @@
...
+appveyor.yml:
+  matrix:
+    - PUPPET_GEM_VERSION: 3.7.5
+      RUBY_VER: 193
+    - PUPPET_GEM_VERSION: 3.7.5
+      RUBY_VER: 200-x64
+    - PUPPET_GEM_VERSION: 3.8.1
+      RUBY_VER: 193
+    - PUPPET_GEM_VERSION: 3.8.1
+      RUBY_VER: 200-x64
```
Comment (ferventcoder) [inline config_defaults.yml]: Generally it's oldest support, current 3.x series and current 4.x series.

File: moduleroot/.travis.yml [hunk 2]
```diff
@@ -16,5 +16,8 @@ matrix:
     env: <%= extra['env'] %>
 <%   end -%>
 <% end -%>
+  allow_falures:
+    - rvm: 2.1.5
+      env: PUPPET_GEM_VERSION="~> 4.0"
```
Comment (ferventcoder) [inline moduleroot/.travis.yml]: This is probably okay...

File: moduleroot/Gemfile [hunk 3]
```diff
@@ -46,6 +46,21 @@ end
...
+  gem "win32-eventlog", "~> 0.5", :require => false
+  gem "win32-process", "~> 0.6", :require => false
+  gem "win32-security", "~> 0.1", :require => false
+  gem "win32-service", "~> 0.7", :require => false
+  gem "win32-taskscheduler", "~> 0.2", :require => false
+  gem "win32console", "~> 1.3", :require => false
+  gem "windows-api", "~> 0.4", :require => false
+  gem "windows-pr", "~> 1.2", :require => false
+  gem "minitar", "~> 0.5.4", :require => false
+end
```
Comment (ferventcoder) [inline moduleroot/Gemfile]: The above is close, I hope you found the updates you needed to make here.

**Pull request merged**